src/controllers/controller.bar.js

```diff
@@ -328,18 +328,13 @@ module.exports = function(Chart) {
 			var me = this;
 			var chart = me.chart;
 			var rects = me.getMeta().data;
-			var dataset = me.getDataset();
 			var ilen = rects.length;
 			var i = 0;
-			var d;
 
 			helpers.canvas.clipArea(chart.ctx, chart.chartArea);
 
 			for (; i < ilen; ++i) {
-				d = dataset.data[i];
-				if (d !== null && d !== undefined && !isNaN(d)) {
```

Comment:
I think the current behaviour for bars is that `null/undefined/NaN` are supposed to just not draw anything but I'm not 100% sure on that. I know we use those values to skip drawing grid lines in the axis though